src/Contracts/Models/ExecutionContext.cs

```diff
@@ -1,38 +1,41 @@
-﻿using System.Collections.Generic;
-using Newtonsoft.Json;
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
 
 namespace Monai.Deploy.WorkflowManager.Contracts.Models
 {
     public class ExecutionContext
     {
-        [JsonProperty(PropertyName = "execution_id")]
+        [ConfigurationKeyName("execution_id")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
changed